product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+
+[Managing service limits and constraints in Azure](01a_managing_quotas) describes how to check that your clusters meet these quotas and how to raise your resource limits, if needed.
+
+### IP Address quota
+
+You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your clusters with the total number of clusters you plan to use. To understand the requirements, and recommended quotas, see the following sections.
+
+**Default quota**
+
+The default public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+
+**Recommended quota**
+
+EDB recommends the following when requesting public IP address quota increases:
+
+* Public IP Addresses – Basic: number of clusters you plan to deploy in a given region
+* Public IP Addresses – Standard: number of clusters you plan to deploy in a given region
+
+### vCPU quota
+
+Any time a new VM is deployed the vCPUs for the VMs must not exceed the vCPU quota for the VM size family or the total regional vCPU quota. The VMs deployment fails if either of those quotas are exceeded. 
+
+!!! Note
+    The default number of total cores per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information.
+
+In each region, EDB Cloud uses six ESv3 and six DSv2 virtual machine cores to manage your EDB Cloud infrastructure.
+Your clusters deployed in the region use separate ESv3 virtual machine cores. The number of cores depends on the Instance Type and High Availability options of the clusters you provision. You can calculate the number of ESv3 cores required for your cluster based on the following:
+
+* Virtual machine instance of type E{N}Sv3 uses {N} cores. For example, an instance of type E64Sv3 uses 64 ESv3 cores.
+* Cluster running on an E{N}Sv3 instance with high availability not enabled uses exactly {N} ESv3 cores.
+* Cluster running on an E{N}Sv3 instance with high availability enabled uses 3 * {N} ESv3 cores.
+
+As an example, if you provision the largest virtual machine E64Sv3 with high availability enabled, it requires (3 * 64)=192 ESv3 cores per region. EDB Cloud infrastructure requires an additional six ESv3 and six DSv2 virtual machine cores per region.
+
+**Recommended quota**
+
+EDB recommends the following when requesting Virtual Machine resource quota increases:
+* Total Regional vCPUs: minimum of 100+ per designated region
+* Standard ESv3 Family vCPUs: minimum of 50+ per designated region
+* Standard DSv2 Family vCPUs: minimum of 20+ per designated region
```

Comment:
Why do we recommend 20?

Reply:
@ba-edb does it depend on the average workload of the customer? what is a good recommendation?

Reply:
No, it doesn't depend on it very closely - we just need 6 DSv2 to start with. But they might need more in the future, and we're going to be changing it... I think this okay for now.